crates/conversions/tests/helpers/hex.rs

```diff
@@ -0,0 +1,39 @@
+use cairo_felt::Felt252;
+use starknet::core::types::FieldElement;
+use starknet_api::hash::StarkFelt;
+
+#[must_use]
+pub fn str_hex_to_stark_felt(hex_string: &str) -> StarkFelt {
+    let hex_string = hex_string.strip_prefix("0x").unwrap().to_string();
+    let mut byte_array = [0u8; 32];
+    for i in 0..32 {
+        let byte_str = &hex_string[i * 2..i * 2 + 2];
+        byte_array[i] = u8::from_str_radix(byte_str, 16).unwrap();
+    }
+
+    StarkFelt::new(byte_array).unwrap()
+}
```

Comment:
All those 3 functions are almost the same. It would be good to extract duplicated code

Reply:
fixed